ECIPs/ECIP-1060.md

```diff
@@ -0,0 +1,413 @@
+ * **`nonce`**: The signer proposal regarding the account defined by the `beneficiary` field.
+   * It should be **`NONCE_DROP`** to propose deauthorizing `beneficiary` as an existing signer.
+   * It should be **`NONCE_AUTH`** to propose authorizing `beneficiary` as a new signer.
+   * It **must** be filled with zeroes on checkpoint (i.e., on epoch transition) blocks.
+   * It **must** not take up any other value apart from the two above.
+ * **`extraData`**: Combined field for signer vanity, checkpointing and signer signatures.
+   * The first **`EXTRA_VANITY`** fixed bytes may contain arbitrary signer vanity data.
+   * The last **`EXTRA_SEAL`** fixed bytes are the signer's signature sealing the header.
+   * Checkpoint blocks **must** contain a list of signers (`SIGNER_COUNT*20` bytes) in between, **omitted** otherwise.
+   * The list of signers in checkpoint block extra-data sections **must** be sorted in ascending order.
+ * **`mixHash`**: Reserved for fork protection logic, similar to the extra-data during the DAO.
+   * It **must** be filled with zeroes during regular operation.
+ * **`ommersHash`**: It **must** be **`UNCLE_HASH`** as uncles are meaningless outside of proof-of-work.
+ * **`timestamp`**: It **must** be greater than the parent timestamp + **`BLOCK_PERIOD`**.
+ * **`difficulty`**: It contains the standalone score of the block to derive the quality of a chain.
+   * It **must** be **`DIFF_NOTURN`** if `BLOCK_NUMBER % SIGNER_COUNT != SIGNER_INDEX`
+   * It **must** be **`DIFF_INTURN`** if `BLOCK_NUMBER % SIGNER_COUNT == SIGNER_INDEX`
+
+### Validator List
+
+The _initial_ validator list can be specified in the configuration at genesis, i.e., by appending it to the Clique config in Geth:
+
+```json
+"clique":{
+  "period": 15,
+  "epoch": 30000,
+  "validators": [
+    "0x7d577a597b2742b498cb5cf0c26cdcd726d39e6e",
+    "0x82a978b3f5962a5b0957d9ee9eef472ee55b42f1"
+  ]
+}
+```
+
+By using this list, for convenience, the `extraData` field of the genesis block only has to contain the 32 bytes of **`EXTRA_VANITY`**. The client automatically converts and appends the list of signers to the block `0` extra-data as specified in EIP-225 at checkpoint blocks (appending `SIGNER_COUNT*20` bytes).
+
+### Sealing
+
+For a detailed specification of the block authorization logic, please refer to EIP-225 by honoring the constants defined above. However, the following changes should be highlighted:
+
+* Each singer is **allowed to sign any number of consecutive blocks**. The order is not fixed, but in-turn signing weighs more (**`DIFF_INTURN`**) than out-of-turn one (**`DIFF_NOTURN`**). In case an out-of-turn block is received, an **in-turn signer should continue to publish their block** to ensure the chain always prefers in-turn blocks in any case. This strategy prevents in-turn validators from being hindered from publishing their block and potential network halting.
```

Comment:
Yes! We ran into issues of offline validator nodes hindering the whole network, so allowing in-turn signer to continue seems like a blessing.